### PR TITLE
raft-ann-bench package fixes for plotting and conf files

### PR DESCRIPTION
**Pull request by dantegd**
Number of improvements and fixes for different million scale datasets that are supported by default by the python benchmarking package.

### CODE REVIEW

File: python/raft-ann-bench/src/raft-ann-bench/run/conf/nytimes-256-inner.json [hunk 1]
```diff
@@ -0,0 +1,1352 @@
...
+        {"ef":40, "numThreads":1},
+        {"ef":60, "numThreads":1},
+        {"ef":80, "numThreads":1},
+        {"ef":120, "numThreads":1},
+        {"ef":200, "numThreads":1},
+        {"ef":400, "numThreads":1},
+        {"ef":600, "numThreads":1},
+        {"ef":800, "numThreads":1}
+      ],
+      "search_result_file" : "result/nytimes-256-inner/hnswlib/M12"
```
Comment (divyegala) [inline python/raft-ann-bench/src/raft-ann-bench/run/conf/nytimes-256-inner.json:32]: This key is not needed anymore, it was necessary in the old benchmarks

Comment (dantegd) [reply python/raft-ann-bench/src/raft-ann-bench/run/conf/nytimes-256-inner.json:32]: we still have it in a lot of files, perhaps I can do a follow up to remove it everywhere?

**Review by divyegala — APPROVED** [review]
Approved, pending successful local runs

File: python/raft-ann-bench/src/raft-ann-bench/run/conf/nytimes-256-inner.json (hunk 1 above)
Comment (divyegala) [reply python/raft-ann-bench/src/raft-ann-bench/run/conf/nytimes-256-inner.json:32]: Please!

**Pull request merged**